doc/gen_docs.go

```diff
@@ -21,22 +20,15 @@ func main() {
 		"Markdown": genMarkdown,
 	}
 
-	var wg sync.WaitGroup
```

Comment:
why are you removing that? Fast build is nice

Reply:
See the commit message - without this commit I usually get a data race when running `make`.
I didn't really investigate what the real fix should be. Do you think we should use global variables for the Cobra commands?

Reply:
It's the opposite actually. Cobra use a global map [here](https://github.com/spf13/cobra/blob/master/custom_completions.go#L21). This global map is shared even if multiple independent root command are created.

It looks like it could be easily solved in cobra though ...

Reply:
well, not really easy but doable